src/components/FilterGroup.tsx

```diff
@@ -65,6 +72,7 @@ export function FilterGroup({
   showMoreLimit = filterOptions.length,
   children
 }: PropsWithChildren<FilterGroupProps>) {
+  customCssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses);
```

Comment:
generally, for all of our components, we want to call this as low down in the chain of components as possible (so in `CollapsibleLabel` rather than here). this way, the merging of CSS classes only happens once

Reply:
It is already being merged in `CollapsibleLabel`. It was duplicated [here](https://github.com/yext/search-ui-react/blob/39a8996f330c873edf46fbba96208127c07c4b6a/src/components/FilterGroup.tsx#L94) for when `CollapsibleLabel` is disabled, so right now the merging is needed in `FilterGroup` as well. I think a solution to this would be that `CollapsibleLabel` should be able to disable it's collapsing behavior so it can be used to render the label regardless of if `collapsible` is set, but that is out of scope

Reply:
to avoid the multiple merges, could we update it so `useComposedCssClasses` is used in `CollapsibleLabel`, not here, but `twMerge` is used directly for the title `div` in this file (when `CollapsibleLabel` is not being used)?

Reply:
I updated to do this. Perhaps it should be imported directly instead of through index.ts to not expose it so far?

Reply:
are you referring to how `builtInCssClasses` in `CollapsibleLabel ` should be expose? We currently do restrict what is exported out of Filters/index.ts from the top level components folder (in code: https://github.com/yext/search-ui-react/blob/develop/src/components/index.ts#L90-L94) so I think it should be fine leaving `builtInCssClasses` exported through Filters/index.ts and import it like how it's now

Reply:
That sounds good